pkg/devspace/config/loader/validate.go

```diff
@@ -356,9 +359,10 @@ func strMapEquals(a, b map[string]string) bool {
 
 func validateDev(config *latest.Config) error {
 	for index, rp := range config.Dev.ReplacePods {
-		if len(rp.LabelSelector) == 0 && rp.ImageSelector == "" {
-			return errors.Errorf("Error in config: image selector and label selector are nil in replace pods at index %d", index)
+		if rp.ContainerName != "" && len(rp.LabelSelector) == 0 && rp.ImageSelector == "" {
```

Comment:
These need to be separte checks:
1. containerName != "" && len(labelSelector) == 0
2. len(labelSelector) == 0 && imageSelector == ""